contractTest/gateways/MsSQL/verifyUserLogin.contractTest.js

```diff
@@ -0,0 +1,68 @@
+import verifyUserLogin from "../../../src/gateways/MsSQL/verifyUserLogin";
+import AppContainer from "../../../src/containers/AppContainer";
+import setupUser from "../../../test/testUtils/setupUser";
+
+describe("verifyUserLogin contract tests", () => {
+  const container = AppContainer.getInstance();
+
+  xit("verifies if a admin code and password match an existing trust", async () => {
```

Comment:
Out of curiosity what's the reasoning behind using `xit` instead of `it` and same for `xdescribe` in E2E tests?

Reply:
Al tests that require the mssql database have been commented out, because we haven't been able to get the tests running with a mssql database, and they won't pass with a postgres